src/redux/sagas/gigs.saga.js

```diff
@@ -42,11 +42,22 @@ function* postGigs(action) {
     }
 }
 
+function* deleteGig(action) {
+	try{
+		console.log('In saga deleteGig, got the request:', action.payload);
+		yield axios.delete(`api/gig/${action.payload}`)
+		yield put ({type: 'DELETE_GIG'})
```

Comment:
wondering if the type should be FETCH_COMPLETED_GIGS or FETCH_UPCOMING_GIGS instead of DELETE_GIG for the yield put? I think FETCH_UPCOMING_GIGS